### ISSUE TITLE
Remove all references to Applitude

### ISSUE THREAD

**Issue by ericelliott** [ericelliott/cuid#41]
Applitude is defunct. It's past time to remove references to it from Cuid.
- [x] Docs
- [ ] Tests

**Comment by vasco3** [comment]
I'm getting this error when using babel-node
```
> babel-node --ignore '/node_modules/(?!server|client|lib|healthcheck)' app/test/unit/index.js | tap-bail

/var/repositories/universal-react-boilerplate/node_modules/bunyan-request-logger/node_modules/cuid/dist/node-cuid.js:92
})(undefined.applitude || undefined);
            ^
TypeError: Cannot read property 'applitude' of undefined
    at Object.<anonymous> (/var/repositories/universal-react-boilerplate/node_modules/bunyan-request-logger/node_modules/cuid/dist/node-cuid.js:105:3)
    at Module._compile (module.js:460:26)
    at normalLoader (/var/repositories/universal-react-boilerplate/node_modules/babel-core/lib/api/register/node.js:199:5)
    at Object.require.extensions.(anonymous function) [as .js] (/var/repositories/universal-react-boilerplate/node_modules/babel-core/lib/api/register/node.js:216:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/var/repositories/universal-react-boilerplate/node_modules/bunyan-request-logger/request-logger.js:8:10)
    at Module._compile (module.js:460:26)
```
is this related?

**Comment by ericelliott** [comment]
Yes.

**Comment by gjcoombes** [comment]
Hi Eric, PR #53 is a minor tweak that fixed things for me.  Happy to improve the changes as you see fit.

**Comment by viveleroi** [comment]
Can you please release a new NPM version with this fix, we need this as well. Thanks

**Comment by kirilpopov** [comment]
Can you publish new NPM version - we're dropping this wonderful package from our project , because the applitude thing is breaking our build with webpack. Thanks

**Comment by ericelliott** [comment]
The current version is not publishable. It has not been properly tested and will likely break existing projects that depend on it. When the tests can run and pass in browsers, I will publish.

PRs are welcome. See [this issue](https://github.com/ericelliott/cuid/issues/42) for context.

**Comment by Flet** [comment]
Hello!

It looks like some awesome work was done in #72 to resolve this issue and a new version was created here in GitHub, but v2.0.0 was never published to npm.

Is it possible to publish v2.0.0 to npm?

Are there still reservations around breaking projects with this version? If so, is it possible to publish under a tag to allow folks to try it? (i.e. `npm publish --tag beta`)

**Comment by ericelliott** [comment]
Published as v2.0.1
